chat.js

```diff
@@ -2,34 +2,23 @@ import React from "react";
 import ReactDOM from "react-dom";
 import route from "can-route";
 import DefineMap from "can-define/map/";
-import Component from "react-view-model/component";
 import Messages from "./components/messages";
+import { connect, ObserveObject } from "ylem";
 
-export const ViewModel = DefineMap.extend('AppVM', {
-	page: "string",
-	message: {
-		type: "string",
-		value: "Chat Home",
-		serialize: false
-	}
-});
+class Store extends ObserveObject { 
```

Comment:
same as other repo, probably use `ChatStore` here

Reply:
This store looks empty (though can-route will be adding things). I like to decide "defaults" to the things I will be adding later (in leiu of the definitions from can-define). In this case, it is adding `page` so I would add `page = null` or `page = ""` or something to the class.